EFDataAccessLibrary/Repositories/ContactRepository.cs

```diff
@@ -22,30 +22,46 @@ public void AddRange(IEnumerable<Contact> contacts)
 			_db.AddRange(contacts);
         }
 
+        public async Task AddRangeAsync(IEnumerable<Contact> contacts, CancellationToken cancellationToken)
+        {
+			await _db.AddRangeAsync(contacts, cancellationToken);
+        }
+
         public List<Contact> GetAllContacts()
```

Comment:
das bruchts au nüm oder? git ja jetzt GetAllContactsAsync

Reply:
> das bruchts au nüm oder? git ja jetzt GetAllContactsAsync

Das wird no benutzt in LoadSampleData() für de fall dass mySQL DB no kei date het in index.cshtml.cs line 48 "            await _db.AddRangeAsync(contacts, cancellationToken);"